config/model_configs/les_isdac_box.yml

```diff
@@ -0,0 +1,32 @@
+# device: CPUSingleThreaded
+job_id: "les_isdac_box"
+initial_condition: "ISDAC" 
+subsidence: "ISDAC" 
+surface_setup: "ISDAC"
+external_forcing: "ISDAC"
+rad: "ISDAC"
+moist: "equil"
+config: "box" 
+hyperdiff: "true" 
```

Comment:
We can also set this to `false` and remove the `coefficient` values below to reduce the `yaml` file size.